CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project are documented in this file.
 
+## 14.10.0
+
+- Added: `reference` export, which contains reference data from the CSS spec that may be useful for custom lints.
+
```

Comment:
[suggest] Could you revert the change, please? We will update the changelog after merged.

```suggestion
```